src/components/ImageUploader/src/ImagePicker.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+	<div
+		:class="$s.ImagePickerInputContainer"
+		@click="$refs.input.click()"
+	>
+		<input
+			ref="input"
+			:class="$s.ImagePickerInput"
+			type="file"
+			multiple="true"
```

Comment:
I think this should be removed and let the user pass it in via `v-bind="$attrs"`. eg. What if the use-case only allows one image?

Reply:
That, or since we have `max-images`, we set this to be `true` when that is set `>1`, then `false` if set to `1`